src/clnt_internal.h

```diff
@@ -65,4 +65,16 @@ clnt_data_destroy(struct cx_data *cx)
 		mem_free(cx->cx_c.cl_tp, strlen(cx->cx_c.cl_tp) + 1);
 }
 
+
+/*
+ * Make sure that the time is not garbage.  0 or - value is *NOT* allowed.
+ */
+static inline bool
+time_not_ok(struct timeval *t)
+{
+	return (t->tv_sec < 0 || t->tv_sec > 99 ||
```

Comment:
I disagree with this.  The caller should be able to pick a large timeout, if they want.  99 it much to arbitrarily small.  For example, the TCP timeout is 60 minutes.  Leave it large.